deployments/relayer/build-path

```diff
@@ -27,6 +27,6 @@ visible_log "These commands should all yield results:"
 
 cat <<EOF
   * pcli -n http://localhost:8080 q ibc client 07-tendermint-0
-  * pcli -n http://localhost:8080 q ibc channel transfer channel-0
+  * pcli -n http://localhost:8080 q ibc channel --channel-id 0
```

Comment:
why does this command have different syntax to all of the other `q ibc` commands?

Reply:
We haven't been exercising these paths much! That's changing now, as IBC comes into focus.